next/blog/components/layout/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import NextLink from 'next/link'
+import { BCMSImage } from "~~/bcms-components";
+import classNames from "classnames";
+import {ContentManager} from "~/components/ContentManager";
+import { HeaderEntryMeta } from "~~/bcms/types";
+import { useState} from "react";
+import {MenuIcon} from "@/assets/icons/menu";
+import {XIcon} from "@/assets/icons/x";
+import {TopGradient} from "~/components/TopGradient";
+
+export function Header ({data}: {data:HeaderEntryMeta }): JSX.Element {
+    const [showMobileMenu, setShowMobileMenu] = useState<boolean>(false)
+    return (
+        <header className="relative z-50">
+            <div className="relative z-10 container">
+                <nav className="relative flex items-center justify-between pt-6">
+                    <NextLink href="/" className="flex" passHref={true}>
+                       <a>
+                           <BCMSImage
+                               media={data.logo}
+                               svg
+                               className={classNames('w-[125px] md:w-[165px]', showMobileMenu && 'max-md:grayscale max-md:brightness-0 max-md:invert' )}
+                           />
+                       </a>
+                    </NextLink>
+                    <ul
+                        className={classNames('flex gap-6 max-md:absolute max-md:left-0 max-md:-bottom-6 max-md:translate-y-full max-md:w-full md:flex-row md:justify-center lg:gap-10',  showMobileMenu ? 'flex-col' : 'max-md:hidden')}
+                    >
+                        {data.nav.map((item, index) => (
+                            <li key={index}>
+                                <ContentManager
```

Comment:
Header links are opening in new tab. They should only change window route using NextLink.